### ISSUE TITLE
Frases deseadas en text search

### ISSUE THREAD

**Issue by furzuadr** [IIC2413/Syllabus-2020-1#343]
Hola!
No entendemos a qué se refiere con "frase deseada" dentro del text search, qué tiene que pasar?
Gracias!

**Comment by nivek0o0** [comment]
Tienen que retornar los mensajes que contengan **al menos una** de las palabras de entre todas las frases deseadas. Es un OR lógico entre las palabras deseadas.

En la [documentación](https://docs.mongodb.com/manual/text-search/) ejemplifican los 3 casos:

- Frases deseadas: lo que pasa en la sección de `$text Operator`
- Frases requeridas: lo que pasa en la sección `exact phrase`
- Palabras prohibidas: lo que pasa en la sección `term exclusion`

**Comment by jjaubele** [comment]
Entonces @nivek0o0, si por ejemplo desired = ["En Chile", "Vivo yo"] ¿basta con que aparezca solo "En" o "yo"  en un mensaje para que ese mensaje se retorne como mensaje  deseado?

**Comment by nivek0o0** [comment]
Sí, porque es justo así como funciona la búsqueda de texto en Mongo y lo que queremos que implementen.
